docs/tables/mongodbatlas_database_user.md

```diff
@@ -0,0 +1,28 @@
+# Table: mongodbatlas_database_user
+
+A database user has access to databases in a mongodb cluster. Each user has a set of roles that provide access to all databases in the project.
+
+## Example
+
+### Basic Info
+
+```sql
+select
+  id,
+  name
+from
+  mongodbatlas_database_user
+```
+
+### List all database users who have 'readWriteAnyDatabase' role on the database 'admin'
+
+```sql
+select
+  username
```

Comment:
Should we also include the database name (or other columns), so when users look at the results, they know what database the user is in?